src/rtc.rs

```diff
@@ -46,6 +46,14 @@ pub enum Error {
     InvalidInputData,
 }
 
+/// Low Speed clock to be used by the RTC.
+pub enum LSClock {
```

Comment:
Would it make more sense to call this `RtcClockSource`?

```suggestion
/// Low speed clock source to be used by the RTC.
pub enum RtcClockSource {
```

Reply:
Chose to rename it just `ClockSource` to avoid duplicated use of `Rtc` as in `rtc::RtcClockSource`